java/src/jmri/jmrit/beantable/LightTableAction.java

```diff
@@ -2310,8 +2314,23 @@ public boolean isValid() {
             } else if ((allow0Length == true) && (value.length() == 0)) {
                 return true;
             } else {
-                return true; // TODO temporarily disabled checking format while adding user feedbac
-                // return InstanceManager.getDefault(LightManager.class).validSystemNameFormat(prefix + "L" + value); // get prefixSelectedItem
+                // store previous level
+                Level prevLogLevel = org.apache.log4j.Logger.getRootLogger().getLevel();
```

Comment:
Thanks, I will think of another solution to prevent the console getting filled with validation lines. That's not really a bug or an issue, just clutter.
The plan is to present this as feedback to the user via the status bar or a dialog anyway, using a special EntryException, and display that in the GUI instead of the console.
Once that's in place, would it be OK to go through all the managers and lower these warns to debug level?

Reply:
The underlying problem, I.e. The right thing to fix, is that the *Action code is calling the validation in violation of contract: validation is meant to be of _complete_ names. 

If partial validation is needed, let's change the return value to an enum with "OK", "FAIL" and "OK_AS_START" as possible values.

Reply:
First off, my lesson here is: Fix the underlying problem, not manage the damage.
Applied to this case, it could be fixed either as suggested or dig even deeper: what is needed on the pane is a check of the second name part, and the initial approach failed as I added a regex that essentially duplicated what (part of) ``ValidSystemNameFormat()`` already provided. This is also evident in a lot of the *Address methods; they show a lot of duplication in their checks. May I suggest to separate the checking of the "hardware address" (third part of system name) from VSNF() into ``ValidHardwareAddress(String addr)``, call it directly for the entry pane check and as part of VSNF() and apply the enum return value/exception feedback to allow each calling method to handle it as wanted (loud vs. quietly)?